main.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const mltWrapp = document.getElementById('mltWrapp');
+const fzWrapp = document.getElementById('fzWrapp');
+const fzTitle = document.getElementById('fzTitle');
+const bzWrapp = document.getElementById('bzWrapp');
+const bzTitle = document.getElementById('bzTitle');
+
+mltWrapp.style.background = 'yellow';
+fzWrapp.style.background = 'pink';
+fzTitle.style.color = 'tomato';
+bzWrapp.style.backgroundColor = 'skyblue';
+bzTitle.style.color = 'white';
+
+const onPress = () => {
+    let min = document.getElementById('minNum').value;
+    let max = document.getElementById('maxNum').value;
+    console.log(min);
+    console.log(max);
+
+    let fzInput = document.getElementById('fzInput').value;
+    let bzInput = document.getElementById('bzInput').value;
+    let fbzResult = document.getElementById('fbzResult');
+
+    let fbzCheck1 = /^\D+$/;             // 整数であることのチェック
+    let fbzCheck2 = /^-?[0-9]+\.[0-9]+$/ // 小数点が含まれているかのチェック
+
+    if (min >= max) {
+        fbzResult.innerHTML = '';
+        fbzResult.innerHTML = '倍数を[Min > Max]となるように入力してください';
```

Comment:
エラーチェックのコードは以下のようにすぐに `return` させる方が可読性が高くなります。

```suggestion
    if (min >= max) {
        fbzResult.innerHTML = '';
        fbzResult.innerHTML = '倍数を[Min > Max]となるように入力してください';
        return;
    }
```